ammo.idl

```diff
@@ -576,6 +576,9 @@ interface btDiscreteDynamicsWorld {
   void removeConstraint(btTypedConstraint constraint);
 
   long stepSimulation(float timeStep, optional long maxSubSteps, optional float fixedTimeStep);
+  void addVehicle (btActionInterface vehicle);
+  void removeVehicle (btActionInterface vehicle);
```

Comment:
These functions are obsolete. You're supposed to use add/removeAction instead:

http://bulletphysics.org/Bullet/BulletFull/classbtDynamicsWorld.html#ab897455739081304249e7af787cafa19